tests/index.test.js

```diff
@@ -0,0 +1,26 @@
+const { expect } = require('chai')
+const { it, describe } = require('mocha')
+const { notEqual }  = require('../index')
+
+
+
+  describe('Comparisons', () => {
+
+    it('should return true when comparing 2 numbers', () => {
+      const val1 = 10
+      const val2 = 10
+      const result = notEqual(val1, val2)
+
+
+      expect(result).to.equal(true)
+    })
+
+    it('should return false when comparing number and a string', () => {
+      const val1 = 10
+      const val2 = '10'
+      const result = notEqual(val1, val2)
+
+
+      expect(result).to.equal(false)
+    })
+  })
```

Comment:
Make more tests to check for the same data type but different values for example (5, 10) or ('a', 'b')